ctapipe/core/tests/test_feature_generator.py

```diff
@@ -51,3 +51,19 @@ def test_missing_colname():
 
     with pytest.raises(ExpressionError):
         generator(table)
+
+
+def test_to_unit():
+    """Test chaning the unit of a feature"""
+    from astropy import units as u
+
+    expressions = {
+        "length_meter": "length.to(u.m)",
+        "log_length_meter": "log10(length.to(u.m).value)",
```

Comment:
to_value

Reply:
Not possible on Columns:
```python
>>> from astropy import units as u
>>> from astropy.table import Table

>>> table = Table({'length': u.Quantity([1, 2, 3], unit=u.m)})
>>> table['length'].to_value(u.km)

Traceback (most recent call last):
  File "column_to_value.py", line 6, in <module>
    table['length'].to_value(u.km)
AttributeError: 'Column' object has no attribute 'to_value'
```

Reply:
.quantity.to_value

Reply:
Would be possible on `QTable` though, but this is not what we use internally

Reply:
thanks